client/state/directly/actions.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * Internal dependencies
+ */
+import {
+	initialize as initializeDirectly,
+	askQuestion as askDirectlyQuestion,
+	maximize as maximizeDirectly,
+	minimize as minimizeDirectly,
+	openAskForm as openDirectlyAskForm,
+} from 'lib/directly';
+import {
+	DIRECTLY_ASK_QUESTION,
+	DIRECTLY_INITIALIZE,
+	DIRECTLY_MAXIMIZE,
+	DIRECTLY_MINIMIZE,
+	DIRECTLY_OPEN_ASK_FORM,
+} from 'state/action-types';
+
+export const askQuestion = ( questionText, name, email ) => ( dispatch ) => {
+	dispatch( { type: DIRECTLY_ASK_QUESTION, questionText, name, email } );
+	askDirectlyQuestion( questionText, name, email );
+};
```

Comment:
well truth be told I've carried the banner for the fight against them but that's because of the damage they do to our ability to introspect, modify, and centrally manage our app policies and behaviors. they are convenient for local concerns but since they fly through the dispatcher hiding what could be any code imaginable they block our ability to perform meta operations on the actions, orient our code to component-specific changes instead of app-specific intentions, and present a mess for maintaining the hundreds of related thunk action types that exist in the  Calypso code nebula